HTML/index.html

```diff
@@ -3,44 +3,219 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Movie website Home Page</title>
-  <link rel="stylesheet" href="../CSS/index.css">
-
+  <title>Movie</title>
+  <link rel="stylesheet" href="../CSS/style.css">
+  <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
 </head>
 <body>
-  <!-- Header Section -->
   <header>
-    <nav>
-      <!-- nav bar logo -->
-      <div class="logo">
-        <h1>Movie<span>360</span></h1>
-      </div>
-
-      <!-- Nav items: -->
-      <div class="nav-items">
+    <div class="logo">
+      <h1>Movie<span>360</span></h1>
+    </div>
+    <span class="close" id="close">&#10006;</span>
+    <span id="bars">
+      <i class='bx bx-menu'></i>
+    </span>
+    <nav id="nav">
+      
+      <div class="nav-items" id="nav-items">
```

Comment:
Your navbar looks good but try to use nav-items in same container for both desktop and mobile.